proxies/live/apiproxy/resources/jsc/ConnectionMetadata.SetRequestHeaders.js

```diff
@@ -49,11 +49,24 @@
         "request.header.NHSD-End-User-Organisation"
     );
 
+    var nrlPermissions = context.getVariable('app.nrl-permissions');
+
+    // Convert it into a complex object
+    var permissionLines = nrlPermissions.split(/\s+/);
```

Comment:
Sorry to be a pain but do we know the behaviour here if there are no permissions available on the app? Will this bomb out or just continue?